dappnode_package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hopr.public.dappnode.eth",
-  "version": "1.0.27",
+  "version": "1.0.19",
   "upstreamVersion": "providence",
```

Comment:
Upstream version should be `saint-louis`

Reply:
Good point, will change it.